tests/integration/solidity_contracts/PlainOpcodes/test_safe.py

```diff
@@ -1,38 +1,32 @@
 import pytest
+from tests.utils.constants import ACCOUNT_BALANCE
 
 
 @pytest.mark.asyncio
 @pytest.mark.Counter
```

Comment:
can you fix this copy/pasted mark also plz?

Reply:
done.